File: Solutions/python/SetupSolutions/SetupSolutions.py
import os

def getPuzzleInput(year: str, day: str):
    inputPath = os.path.join("..", "..", "..", "Inputs", year, day)
    scriptInDirectory = os.path.dirname(os.path.realpath(__file__))
    puzzlePath = os.path.join(scriptInDirectory, inputPath)

    with open(puzzlePath) as file:
        puzzle = file.readlines()
    
    return puzzle

File: Solutions/Golang/SetupSolutions.go
package main

import (
	"fmt"
	"log"
	"os"
	"path/filepath"
)

func getPuzzleInput(year string, f string) {
	file := filepath.Join("..", "..", "Inputs", year, f)
	content, err := os.ReadFile(file)
	if err != nil {
		log.Fatal(err)
	}
	fmt.Println(string(content))
}
